documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -284,19 +284,20 @@ If your metadata processors create fields with a different name from the one pas
 === Queries
 
 You can write queries like you usually do in Hibernate Search: native Lucene queries and DSL queries (see <<search-query>>).
-We do automatically translate the most common types of Apache Lucene queries and many of the queries generated by the Hibernate Search DSL.
+We do automatically translate the most common types of Apache Lucene queries
+and all but one the query generated by the Hibernate Search DSL (see below).
 
 [NOTE]
 .Unsupported Query DSL features
 --
 Queries written via the DSL work.
 Open a JIRA otherwise.
 
-Notable exceptions are:
+The notable exception is more like this queries.
```

Comment:
should we quote or italicize "more like this" queries? The sentence seems weird, if you don't already know in advance what you mean.